client/state/selectors/get-site-editor-url.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { shouldRedirectGutenberg } from 'state/selectors/should-redirect-gutenberg';
+import { getSiteAdminUrl, getSiteSlug } from 'state/sites/selectors';
+
+/**
+ * Retrieves url for site editor.
+ *
+ * @param {object} state  Global state tree
+ * @param {object} siteId Site ID
+ * @returns {string} Url of site editor instance for calypso or wp-admin.
+ */
+export const getSiteEditorUrl = ( state, siteId ) => {
+	const siteAdminUrl = getSiteAdminUrl( state, siteId );
+
+	// I'm not sure if we will want this here?
+	if ( shouldRedirectGutenberg( state, siteId ) ) {
```

Comment:
I think we only need this for some cases in which we cannot load the remote page through the iFrame (might be some security checks too; I'm not sure.) So it's probably fine to keep it since the same logic probably applies